CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Drafter Changelog
 
+## master
+
+* Add column/line info to anotations source maps 
+* change start column index from 1 to 0 in reporting. (while `-l` switch is used)
```

Comment:
Why is this done? This is UI. We need to use index starting from 1. @kylef What do you think?

Reply:
https://github.com/apiaryio/drafter/pull/653/files/3217ac2c099085b7bfd2640e1ad91751287697e2#r223059516

Reply:
I think you mean the `-u` flag? 

>   -u, --use-line-num    use line and row number instead of character index when printing annotation

I don't think this is nessecerily done in the UI, it is an explanation that `-u` flag will now behave differently than before because the same logic is used for column/line numbers.

Reply:
> https://github.com/apiaryio/drafter/pull/653/files/3217ac2c099085b7bfd2640e1ad91751287697e2#r223059516

Based on my reading of the spec Pavan raised a valid point. There is actually no need to change to zero-based indexing for column and line attributes of source maps; I initially also misread the spec, because it talks about zero-based indexing - of the byte offset however, not column and line numbers. All in all, column and line numbers are good old natural numbers, and I would keep that. Natural numbers, in my world, start from 1.

Reply:
Ok, so do you want to return back implementation to one-indexed like in original code?

I made unit tests related to indexing 25 days ago. It was clear from this code how indexing will be implemented. I asked for review, nobody commented it...
https://github.com/apiaryio/drafter/pull/653/commits/f255865e30db7ef40c1ef55d7681858b690e1f02